src/js/controllers/index.js

```diff
@@ -140,27 +153,30 @@ angular.module('copayApp.controllers').controller('indexController', function($r
       }, 300);
     }
 
-    if (!self.tab)
+    if (!self.tab || !$state.is('walletHome'))
       self.tab = 'walletHome';
 
-    if (document.getElementById(self.tab)) {
-      document.getElementById(self.tab).className = 'tab-out tab-view ' + self.tab;
-      var old = document.getElementById('menu-' + self.tab);
-      if (old) {
-        old.className = '';
+    go.path('walletHome', function() {
```

Comment:
Why you removed go.path()? IIUC, go.path('walletHome', is needed for setting the state to `walletHome` when / if set path is called from outside a inner tab.